app/OllamaService.php

```diff
@@ -3,31 +3,43 @@
 namespace App;
```

Comment:
After reviewing the file app/OllamaService.php, I found the following issues:

- The class does not extend a parent class, which could cause unexpected behavior when using the class.

- There are no validation checks on the parameters passed to the methods of this class, which could lead to security vulnerabilities or incorrect results if the input is not properly validated. 

LGTM